Modules/Module_Import/xASL_wrp_DCM2NII_Subject.m

```diff
@@ -43,21 +43,14 @@
     % Check subjectID
     thisSubject.subjectExport = xASL_imp_SubjectName(subjectID);
     
+	x.modules.import.imPar.visitNames = thisSubject.visitIDs;
+
     %% 2. Iterate over visits
     for iVisit=1:thisSubject.nVisits
         
         % Get the current visit and the visit ID
         vFieldName = ['visit_' num2str(iVisit,'%03.f')];
         thisVisit = thisSubject.(vFieldName);
-        visitID = thisSubject.visitIDs{iVisit};
-
```

Comment:
Why are you removing all this code? I would still keep the option (defaulting to false) to convert visits to indices, unless this would get messy.

Not sure what functionality you are removing now, it could of course be that you see that you can simplify things but haven't accounted for all use cases?

Reply:
Related to the comment above. This was a pure Michael nonsense that had no function and was just cluttering the code. I'm simply cleaning.

This specifically - he took the visit token. Renamed this to a visit name/number, saved this into his huge structure. And then a few moments later he does the same action again :) 

So I'm just removing this stupid duplicity. I know it's hard to check, but don't worry, I was very carefully reviewing and identifying this nonsense and it made no sense to keep it.